Deps issue when building packages

Checking the logs I have identified an issue happening during `yarn packages:build` or more exactly `yarn workspace @unlock-protocol/contracts build` 

It was introduced with the `hardhat-helpers` package.

No big deal for now but will prevent from updating docs I think

```
 yarn build
....
➤ YN0000: An unexpected error occurred:
➤ YN0000:
➤ YN0000: Error: Cannot find module '/Users/clement/Dev/unlock/unlock/node_modules/@unlock-protocol/networks/dist/index.js'. Please verify that the package.json has a valid "main" entry
➤ YN0000:     at tryPackage (internal/modules/cjs/loader.js:321:19)
➤ YN0000:     at Function.Module._findPath (internal/modules/cjs/loader.js:534:18)
➤ YN0000:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:888:27)
➤ YN0000:     at Function.Module._load (internal/modules/cjs/loader.js:746:27)
➤ YN0000:     at Module.require (internal/modules/cjs/loader.js:974:19)
➤ YN0000:     at require (internal/modules/cjs/helpers.js:101:18)
➤ YN0000:     at Object.<anonymous> (/Users/clement/Dev/unlock/unlock/packages/hardhat-helpers/src/networks.js:1:18)
➤ YN0000:     at Module._compile (internal/modules/cjs/loader.js:1085:14)
➤ YN0000:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
➤ YN0000:     at Module.load (internal/modules/cjs/loader.js:950:32) {
➤ YN0000:   code: 'MODULE_NOT_FOUND',
➤ YN0000:   path: '/Users/clement/Dev/unlock/unlock/node_modules/@unlock-protocol/networks/package.json',
➤ YN0000:   requestPath: '@unlock-protocol/networks'
➤ YN0000: }
➤ YN0000: cp: /Users/clement/Dev/unlock/unlock/packages/contracts/docs: No such file or directory
➤ YN0000: mv: rename /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/docs to /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/archive: No such file or directory
➤ YN0000: ./bin/build_docs.sh: line 24: /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/archive/_category_.json: No such file or directory
➤ YN0000: ls: /Users/clement/Dev/unlock/unlock/packages/contracts/docs/PublicLock/IPublicLockV*: No such file or directory
➤ YN0000: ls: /Users/clement/Dev/unlock/unlock/packages/contracts/docs/Unlock/IUnlockV*: No such file or directory
➤ YN0000: mv: rename /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/archive/PublicLock/IPublicLockV.md to /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/PublicLock.md: No such file or directory
➤ YN0000: mv: rename /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/archive/Unlock/IUnlockV.md to /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/Unlock.md: No such file or directory
➤ YN0000: sed: /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/PublicLock.md: No such file or directory
➤ YN0000: sed: /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/Unlock.md: No such file or directory
➤ YN0000: ./bin/build_docs.sh: line 52: /Users/clement/Dev/unlock/unlock/packages/contracts/docs-reorg/archive/_category_.json: No such file or directory
➤ YN0000: unglobed path: docs/_category_.json
➤ YN0000: unglobed path: docs/README.md
➤ YN0000: copy from: docs/_category_.json
➤ YN0000: copy to: dist/docs/_category_.json
➤ YN0000: copy from: docs/README.md
➤ YN0000: copy to: dist/docs/README.md
^C➤ YN0000: The command failed for workspaces that are depended upon by other workspaces; can't satisfy the dependency graph
➤ YN0000: Failed with errors in 31s 320ms
```

--- a/packages/contracts/package.json
+++ b/packages/contracts/package.json
@@ -22,6 +22,7 @@
     "@nomiclabs/hardhat-etherscan": "3.1.0",
     "@primitivefi/hardhat-dodoc": "0.2.3",
     "@unlock-protocol/hardhat-helpers": "workspace:^",
+    "@unlock-protocol/networks": "workspace:^",
     "@unlock-protocol/tsconfig": "workspace:./packages/tsconfig",
     "@unlock-protocol/types": "workspace:^",
     "chai": "4.3.4",

